.github/workflows/ci.yml

```
@@ -41,7 +41,7 @@ jobs:
     strategy:
       matrix:
         platform: [ubuntu-latest, macos-latest, windows-latest]
-        node-version: [16.x, 18.x]
+        node-version: [18.x, 20.x]
```

```suggestion
        node-version: [18.x, 20.x, 22.x]
```

22 will go LTS in a few months, so good to test it already.